<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-FV9RENB" ignored="6" name="Add_016" tests="1" failures="1" timestamp="12 Jun 2020 12:35:25 GMT" time="0.0" errors="0">
  <testcase name="addStaff5" time="0.0" classname="java5.assignment.test.StaffDAOTest">
    <failure type="org.testng.TestNGException" message="
Conversion issue on parameter: salary">
      <![CDATA[org.testng.TestNGException: 
Conversion issue on parameter: salary
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NumberFormatException: For input string: "aaaaaaaaaaa"
at sun.misc.FloatingDecimal.readJavaFormatString(Unknown Source)
at sun.misc.FloatingDecimal.parseFloat(Unknown Source)
at java.lang.Float.parseFloat(Unknown Source)
at java.lang.Float.valueOf(Unknown Source)
... 19 more
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- addStaff5 -->
  <testcase name="addStaff6" time="0.0" classname="java5.assignment.test.StaffDAOTest">
    <ignored/>
  </testcase> <!-- addStaff6 -->
  <testcase name="addStaff3" time="0.0" classname="java5.assignment.test.StaffDAOTest">
    <ignored/>
  </testcase> <!-- addStaff3 -->
  <testcase name="addStaff2" time="0.0" classname="java5.assignment.test.StaffDAOTest">
    <ignored/>
  </testcase> <!-- addStaff2 -->
  <testcase name="addStaff1" time="0.0" classname="java5.assignment.test.StaffDAOTest">
    <ignored/>
  </testcase> <!-- addStaff1 -->
  <testcase name="addStaff7" time="0.0" classname="java5.assignment.test.StaffDAOTest">
    <ignored/>
  </testcase> <!-- addStaff7 -->
  <testcase name="addStaff4" time="0.0" classname="java5.assignment.test.StaffDAOTest">
    <ignored/>
  </testcase> <!-- addStaff4 -->
</testsuite> <!-- Add_016 -->
